app/assets/stylesheets/atoms/_layout.scss

```diff
@@ -35,6 +35,10 @@
   }
 }
 
+.slab--centered {
+  text-align: center;
```

Comment:
there is a `text--centered` class that does this as well. would the atomic way of doing this be to apply that class to the div as well and not create another class?

Reply:
@jessieay yes, I think it makes sense to use the text--centered class. I was referring to the GetCalFresh website that has a `slab--centered` class. But I like reusing the class that we already have. Will update